### PR TITLE
Added custom bitwarden and dmenu arguments, option to output account information to stdout

### PR DESCRIPTION
**Pull request by avalonwilliams**
Added some new options:
* `--stdout`: outputs username and password to stdout, rather than to the clipboard.
* `--url`: filters accounts by url.
* `-l`: sets the dmenu option `-l` for the account and field menus.

The `--stdout` and `--url` options are mostly for scripting, and the -l option is for ajusting the prompts to the user's cosmetic preferences.

### CODE REVIEW

**Comment by andykais** [comment]
Hi there, thanks for opening a PR! These look like handy flags for scripting. Im curious what you plan to do with `--url`, will you somehow open the dmenu while being aware of a browser's open webpage?

On a less conversational topic though, I wonder if it would make sense to make these commands more general. What do you think about instead making two separate options like so:
```bash
--bw-list-args="--url https://google.com"
--dmenu-args="--lines 1"
```

that leaves the door open for those who wish to add more customization (maybe `--folderid` for `bw`)

**Comment by avalonwilliams** [comment]
That seems like a good idea, and I could implement that easily.

Also, I intend to use the `--url` for minimalist scriptable web browsers (eg. uzbl, luakit), and writing bitwarden plugins for them.

**Comment by avalonwilliams** [comment]
I pushed your suggested changes to the master branch on my repo, and I also added a separate argument for the password dmenu and the other dmenu prompts, as the `-l` option looks rather odd on the password prompt.

File: src/schedule-cleanup.js [hunk 1]
```diff
@@ -21,11 +21,16 @@ module.exports = ({ lockBitwardenAfter, clearClipboardAfter, sessionFile }) => {
         console.debug(`${sessionFile} already removed.`)
       }
       bwRun('lock')
-      console.info('bitwarden is locked.')
+      // don't output to stdout if it is being used for reading information
+      if(!stdout) {
```
Comment (andykais) [inline src/schedule-cleanup.js]: this should be handled in `bin/cli.js`. Choosing the stdout flag should disable any logging console.info. That shouldnt be something you need to worry about every time console.info is called. Just handle it similar to how console.debug is set if the `--verbose` flag is used.

```javascript
console.info = args['stdout']
  ? () => {}
  : console.info
```

File: README.md [hunk 2]
```diff
@@ -10,16 +10,23 @@ clipboard.
 ## Usage
 
 ```
-$ bitwarden-dmenu --help
+$ bitwarden-dmemu --help
```
Comment (andykais) [inline README.md]: typo?

**Review by andykais — CHANGES_REQUESTED** [review]
Looks pretty good! I am sad to see the `dmenu -i` flag go, but in loo of the new general `dmenu-args` command, it does make the most sense.

I am requesting a few small changes, just to keep things clean for the future. Address those and I will merge!

**Review by andykais — APPROVED** [review]

**Comment by andykais** [comment]
published in [v1.4.0](https://www.npmjs.com/package/bitwarden-dmenu)

**Comment by avalonwilliams** [comment]
@andykais I made an [autofill script for qutebrowser](https://github.com/AGitBoy/qutebrowser-bitwarden-dmenu) using these changes, so I just want to say thank you for being so helpful with getting this PR merged and thank you for the wonderful job you did in writing this program.

**Comment by andykais** [comment]
😄 You are very welcome! I will definitely give qutebrowser and your plugin a whirl!

**Pull request merged**